### PR TITLE
Punctuation in tutorial.dsc

### PR DESCRIPTION
**Pull request by sibraxlis**
Cleaned up the tutorial a bit, added punctuation and changed some sentence structure/colors.

### CODE REVIEW

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc [hunk 1]
```diff
@@ -15,90 +15,92 @@ tutorial_data:
...
     subtitle: "Go to the server warp: Marketplace"
     message: "<&6>/warp &eto the server warp: Marketplace"
   7:
     hologram:
       - "<&6>Claims"
-      - "<&e>You can claim chunks to different groups"
-      - "<&e>If a group doesn't exist, it be created"
+      - "<&e>You can claim land!"
+      - "<&e>You can also give access to different groups of players"
+      - "<&e>If a group doesn't exist, it will be created"
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:62]: Do you think these lines could benefit from punctuation?

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc [hunk 2]
```diff
@@ -15,90 +15,92 @@ tutorial_data:
...
+      - "<&e>The &2Outer Realms &eare everywhere outside of that."
     particle_guide: true
   10:
     hologram:
-      - "<&7>The World"
-      - "<&e>The <&c>Savage Lands &eare <&c>PvP Enabled"
-      - "<&e>Beware of your encounters here as mobs also spawn."
-      - "<&e>The <&2>Outer Realms <&e>are a safe haven"
-      - "<&e>No mobs will spawn, and PvP is disabled"
+      - "<&d>The World (2)"
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:81]: what's the (2)?

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc [hunk 3]
```diff
@@ -221,4 +223,4 @@ tutorial_events:
     on player exits spawn_cuboid flagged:tutorial:
       - injet tutorial_skipped
       - narrate "<&e>You have exited spawn, and stopped the tutorial."
-      - narrate "<&e>You can restart the tutorial at any time by using <&b>/tutorial"
\ No newline at end of file
+      - narrate "<&e>You can restart the tutorial at any time by using <&b>/tutorial"
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:226]: Could this line benefit from punctuation?

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc (hunk 2 above)
Comment (sibraxlis) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:81]: There's two headings of The World, should i add Pt. 1 Pt. 2?

Comment (BehrRiley) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:81]: Oh, didn't even notice the first, maybe `(1 / 2)` & `(2 / 2)`?

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc (hunk 3 above)
Comment (sibraxlis) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:226]: TBH I didnt even see the stuff at the bottom yet, i was just working on holograms first.

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc [hunk 4]
```diff
@@ -221,4 +223,4 @@ tutorial_events:
     on player exits spawn_cuboid flagged:tutorial:
       - injet tutorial_skipped
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:224]: pls change `injet` to `inject` for Xeane his poor eyes

Comment (sibraxlis) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc]: Done

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc (hunk 2 above)
Comment (sibraxlis) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc]: Done.

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc (hunk 1 above)
Comment (sibraxlis) [reply denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc]: Done.

File: denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc [hunk 5]
```diff
@@ -9,96 +9,98 @@ tutorial_data:
...
     particle_guide: true
   9:
     hologram:
-      - "<&7>The World"
-      - "<&e>The world is split into 2 sections"
-      - "<&e>The &cSavage Lands &eare the inner 20k"
-      - "<&e>The &2Outer Realms &eare everywhere outside of that"
+      - "<&d><&l>The World (1/2)"
+      - "<&e>The world is split into 2 sections<&co>"
+      - "<&e>The &cSavage Lands &eare the inner 20,000 blocks".
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc:76]: Period here is on the wrong side of the quotations.

**Pull request merged**